applications/dashboard/models/class.logmodel.php

```diff
@@ -404,16 +404,21 @@ public function getWhere($where = false, $orderFields = '', $orderDirection = 'a
             unset($where['Operation']);
         }
 
-        $result = Gdn::sql()
-            ->select('l.*')
+        $sql = Gdn::sql();
+        $sql->select('l.*')
             ->select('ru.Name as RecordName, iu.Name as InsertName')
             ->from('Log l')
             ->join('User ru', 'l.RecordUserID = ru.UserID', 'left')
             ->join('User iu', 'l.InsertUserID = iu.UserID', 'left')
-            ->where($where)
-            ->limit($limit, $offset)
-            ->orderBy($orderFields, $orderDirection)
-            ->get()->resultArray();
+            ->where($where);
+
+        if ($orWhere) {
```

Comment:
If we're expecting an array, I think we should explicitly test for an array, instead of any truthy value.